GettingStarted.md

```diff
@@ -121,6 +121,21 @@ All API's are a translation of the API's available to native platforms and follo
 | SetLocationMonitoringEnabled | Enable or disable location monitoring for geofence monitoring | 
 | PushTokenInfo | return a string version of Push Token on all platforms.| 
 
+### Dictionaries
+
+#### Customer Properties Dictionary
+The Customer Properties is passed a IDictionary in this release future release will support an interface to allow for code completion and refactoring.
```

Comment:
- Properties *are* passed *as an* IDictionary. 
- I'd change the second half to be: "In a future release the Localytics SDK will support an interface..."